Mathlib/Algebra/Algebra/Basic.lean

```diff
@@ -437,7 +439,9 @@ theorem coe_linearMap : ⇑(Algebra.linearMap R A) = algebraMap R A :=
 #align algebra.coe_linear_map Algebra.coe_linearMap
 
 instance id : Algebra R R :=
-  (RingHom.id R).toAlgebra
+  { (RingHom.id R).toAlgebra with
```

Comment:
Can you use `where` notation here?

Reply:
Do you want me to use `Tactic.Spread` or just specify by hand?

Reply:
Oh, is `Tactic.Spread` not imported here?

Reply:
You are correct, it is. This is done now.

Reply:
Did you revert it again?

Reply:
Did you revert it again?

Reply:
Accidentally. It is back now.